### PR TITLE
GitBook: [dx_add-own-connectors] 2 pages modified

### PR DESCRIPTION
**Pull request by johnlafleur**
##What

If a developer wants to add their own connectors and use them without contributing to the codebase, there is no simple article to reply to all their questions. This is a suggestion of a new documentation article to address this use case.

This is something we discussed with Sherif in our 1:1 (that doesn't mean he has seen the suggestion beforehand!).

### CODE REVIEW

**Comment by johnlafleur** [comment]
I closed the former pull request as Gitbook messed things up. Now we have something very clean here.

File: docs/integrations/adding-your-own-connectors.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
+# Adding your own connectors
+
+If you'd like to build new connectors and make them part of the pool of pre-built connectors on Airbyte, first a big thank you, and we invite you to check our [contributing guide](https://docs.airbyte.io/contributing/contributing-to-airbyte).
+
+If you'd like to build new connectors, or update existing ones, for your own usage, without contributing to the Airbyte codebase, then that's what open-source is for, and read along.
```
Comment (sherifnada) [inline docs/integrations/adding-your-own-connectors.md:5]:
```suggestion
If you'd like to build new connectors, or update existing ones, for your own usage, without contributing to the Airbyte codebase, read along. 
```

File: docs/integrations/adding-your-own-connectors.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+* [In Python](https://github.com/airbytehq/airbyte/blob/master/airbyte-integrations/connector-templates/python-source/README.md)
+* [Based on Singer Taps in Python](https://github.com/airbytehq/airbyte/blob/master/airbyte-integrations/connector-templates/singer-source/README.md)
+
+#### **Coding new destination connectors:**
+
+* [In Java](https://github.com/airbytehq/airbyte/blob/master/airbyte-integrations/connector-templates/java-destination/README.md)
+
+Please reach out to us if you'd like help developing integrations in other languages. **Indeed, you are able to create integrations using the language you want, as Airbyte connections run as Docker containers.** We haven't built the documentation for all languages yet, that's all. 
+
+## 3. Adding your connectors on our UI to run them
```
Comment (sherifnada) [inline docs/integrations/adding-your-own-connectors.md:32]:
```suggestion
## 3. Adding your connectors in the UI
```

File: docs/integrations/adding-your-own-connectors.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+#### **Coding new source connectors:**
+
+* [In Python](https://github.com/airbytehq/airbyte/blob/master/airbyte-integrations/connector-templates/python-source/README.md)
+* [Based on Singer Taps in Python](https://github.com/airbytehq/airbyte/blob/master/airbyte-integrations/connector-templates/singer-source/README.md)
+
+#### **Coding new destination connectors:**
+
+* [In Java](https://github.com/airbytehq/airbyte/blob/master/airbyte-integrations/connector-templates/java-destination/README.md)
+
+Please reach out to us if you'd like help developing integrations in other languages. **Indeed, you are able to create integrations using the language you want, as Airbyte connections run as Docker containers.** We haven't built the documentation for all languages yet, that's all.
```
Comment (sherifnada) [inline docs/integrations/adding-your-own-connectors.md:30]:
```suggestion
While the guides above are specific to the languages use most frequently to write integrations, **Airbyte integrations can be written in any language**. Please reach out to us if you'd like help developing integrations in other languages.
```

File: docs/integrations/adding-your-own-connectors.md [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+## 2. Developing your own connectors
+
+It's easy to code your own integrations on Airbyte. Here are some links to instruct on how to code new sources and destinations. 
+
+#### **Coding new source connectors:**
+
+* [In Python](https://github.com/airbytehq/airbyte/blob/master/airbyte-integrations/connector-templates/python-source/README.md)
+* [Based on Singer Taps in Python](https://github.com/airbytehq/airbyte/blob/master/airbyte-integrations/connector-templates/singer-source/README.md)
+
+#### **Coding new destination connectors:**
```
Comment (sherifnada) [inline docs/integrations/adding-your-own-connectors.md:26]: There should also be a step before the UI section that says that in order for Airbyte to be able to leverage a custom integration, it should be published to Dockerhub first (or any image hub that Airbyte can access).

File: docs/integrations/adding-your-own-connectors.md [hunk 5]
```diff
@@ -0,0 +1,49 @@
+# Adding your own connectors
```
Comment (sherifnada) [inline docs/integrations/adding-your-own-connectors.md:1]: Should this be renamed `Custom Connectors`? It's less confusing if we can standardize how we refer to features.

**Review by sherifnada — APPROVED** [review]

File: docs/integrations/adding-your-own-connectors.md (hunk 5 above)
Comment (johnlafleur) [reply docs/integrations/adding-your-own-connectors.md:1]: `Custom Connectors` is way better indeed. Will make the change!

File: docs/integrations/adding-your-own-connectors.md (hunk 4 above)
Comment (johnlafleur) [reply docs/integrations/adding-your-own-connectors.md:26]: I just added a step in section 3 for that, as a first step. Let me know if this is good for you.

File: docs/integrations/adding-your-own-connectors.md (hunk 5 above)
Comment (johnlafleur) [reply docs/integrations/adding-your-own-connectors.md:1]: Committed the changes

**Review by sherifnada — APPROVED** [review]

**Pull request merged**